### PR TITLE
Support Ollama 

### PR DESCRIPTION
**Pull request by valentimarco**
# Description

Support of Ollama

Related to issue #487 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -22,6 +22,7 @@ services:
       - SAVE_MEMORY_SNAPSHOTS=${SAVE_MEMORY_SNAPSHOTS:-false}
     ports:
       - ${CORE_PORT:-1865}:80
+      - 5678:5678
```
Comment (nicola-corbellini) [inline docker-compose.yml:25]: can I ask you what this is for?

Comment (valentimarco) [reply docker-compose.yml:25]: Ops-.- this is for the debug plugin

**Review by nicola-corbellini — APPROVED** [review]
To me it looks fine.
I will test it in the weekend and merge it right after

**Comment by valentimarco** [comment]
> To me it looks fine. I will test it in the weekend and merge it right after
I need to refactor the code, so just wait the new version

**Comment by nicola-corbellini** [comment]
ok let me know

**Pull request closed without merging**